build/config/phone/custom-prefs.js

```diff
@@ -2,3 +2,9 @@
 user_pref('devtools.responsiveUI.customWidth', 320);
 user_pref('devtools.responsiveUI.customHeight', 480);
 user_pref('devtools.responsiveUI.currentPreset', 'custom');
+
+user_pref('dom.caches.enabled', true);
```

Comment:
In theory the 'caches' api should be enabled, but I guess it's not a bad idea to specify it.